api_app/connectors_manager/views.py

```diff
@@ -19,14 +20,21 @@ def list(self, request):
 class ConnectorActionViewSet(PluginActionViewSet):
     queryset = ConnectorReport.objects.all()
 
-    def get_object(self, job_id, connector_name) -> ConnectorReport:
+    def get_object(self, job_id, name) -> ConnectorReport:
         try:
             return self.queryset.get(
                 job_id=job_id,
-                connector_name=connector_name,
+                name=name,
             )
         except ConnectorReport.DoesNotExist:
             raise NotFound()
```

Comment:
The implementation of `get_object` is the same in both subclasses, so it can be moved to `PluginActionViewSet`.